hydro_deploy/core/src/ssh.rs

```diff
@@ -298,17 +332,30 @@ impl<T: LaunchedSSHHost> LaunchedHost for T {
                         Duration::from_secs(1),
                     )
                     .await?;
+                // Launch with perf if specified, also copy local binary to expected place for perf report to work
+                let perf_wrapper = if let Some(perf) = perf.clone() {
+                    // Copy local binary to {output_file}.bins/home/{user}/hydro-{unique_name}
+                    let output_file = perf.output_file.to_str().unwrap();
+                    let local_binary = PathBuf::from(format!(
+                        "{output_file}.bins/home/{user}/hydro-{unique_name}"
```

Comment:
Instead of duplicating the path here can we concatenate `binary_path`?

Reply:
Yeah, why are we creating `local_binary`?

Reply:
It's because the perf.data is linking to a binary it expects at that location (and was there on remote but is not there locally)

Reply:
Oh, so scripts that process perf data look at that file to find symbols and what not?

Reply:
Yeah, so if the file isn't there perf shows a bunch of unlabeled functions